Scripts/Levels/Level Main.gd

```diff
@@ -102,6 +108,12 @@ func add_extra_move():
 		curr_moves += 1
 		max_moves += 1
 		update_move_icons()
+		
+func remove_move():
```

Comment:
I think it shouldn't decrease max moves. And if you do apply the change it'd be good to use remove_move() to decrease the move count in the normal course of the game.